scene/gui/code_edit.cpp

```diff
@@ -2180,7 +2136,7 @@ void CodeEdit::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_auto_indent_prefixes"), &CodeEdit::get_auto_indent_prefixes);
 
 	ClassDB::bind_method(D_METHOD("do_indent"), &CodeEdit::do_indent);
-	ClassDB::bind_method(D_METHOD("do_unindent"), &CodeEdit::do_unindent);
+	ClassDB::bind_method(D_METHOD("do_unindent"), &CodeEdit::unindent_lines);
 
 	ClassDB::bind_method(D_METHOD("indent_lines"), &CodeEdit::indent_lines);
 	ClassDB::bind_method(D_METHOD("unindent_lines"), &CodeEdit::unindent_lines);
```

Comment:
What's the point of having both `do_unident` and `unident_lines` exposed but duplicates?

Reply:
Eh, it's an oversight. I don't know how to handle this tbh.
Maybe I should restore `do_unindent()`, but keep `ui_text_dedent` as `unindent_lines()`? I don't even know what is `do_unindent()` for, its behavior is so useless >_>